_pages/software-tools/tock.md

```diff
@@ -95,9 +96,9 @@ You must not bill for the following activities because these are indirect costs
 
 ### Weekly Billable Hour Expectations
 
-If you’re an individual contributor or a lead, you’re expected to spend at least 80% of your time on billable project work. Directors and supervisors have different billability expectations.
+If you’re an individual contributor or a lead, you’re expected to spend at least 90% of your time on billable project work. Directors and supervisors have different billability expectations.
 
-That leaves up to 20% of your time to be spent on non-billable work. There are a few types of non-billable work, used to manage the running of TTS and your part of TTS itself: GSA-mandated non-billable work, 18F business development, and 18F/CoE/etc. non-billable work.
+That leaves up to 10% of your time to be spent on non-billable work. There are a few types of non-billable work, used to manage the running of TTS and your part of TTS itself: GSA-mandated non-billable work, 18F business development, and 18F/CoE/etc. non-billable work.
 
```

Comment:
Should hiring be on this list? Is this meant to apply to other parts of TTS as well? Don't they have different billable expectations than 18F?

Reply:
I updated, let me know what you think

Reply:
👍